app/assets/stylesheets/layout_components/siteheader.scss

```diff
@@ -95,15 +95,17 @@
     padding: 0 $base-padding;
   }
 
+  a{
```

Comment:
Minor, but it might be worth adding a class here to make this selector more specific.